src/Tests/Feature/Web/ThreadStoreTest.php

```diff
@@ -88,7 +83,7 @@ public function should_create_a_post_with_the_thread()
                 'content' => "Thread content"
             ]);
 
-        $this->assertEquals(Thread::count(), 1);
-        $this->assertEquals(Post::count(), 1);
+        $this->assertEquals(1, Thread::count());
```

Comment:
I'd say checking the thread count is unnecessary here - the test is checking whether a post was created.